acer-wmi-battery.c

```diff
@@ -9,8 +9,10 @@
 
 #include <linux/init.h>
 #include <linux/kernel.h>
+#include <linux/limits.h>
 #include <linux/module.h>
 #include <linux/acpi.h>
+#include <linux/unaligned.h>
```

Comment:
I would like to keep this backwards compatible with older kernel versions.
```suggestion
#include <linux/version.h>
#if LINUX_VERSION_CODE < KERNEL_VERSION(6,12,0)
#include <asm/unaligned.h>
#else
#include <linux/unaligned.h>
#endif
```
I looked up the kernel version in which the header location was changed, but I am not quite sure about the last component of the version.

Reply:
Ok, i will add such a check in the next revision.